frontend/src/routes/Governance/discovered/ByCluster/DiscoveredByCluster.tsx

```diff
@@ -0,0 +1,84 @@
+  policies = [],
+  policyKind,
+}: Readonly<{ policies: DiscoveredPolicyItem[] | undefined | null; policyKind: string }>) {
+  const { t } = useTranslation()
+  const { channelsState, helmReleaseState, subscriptionsState } = useSharedAtoms()
+  const helmReleases = useRecoilValue(helmReleaseState)
+  const subscriptions = useRecoilValue(subscriptionsState)
+  const channels = useRecoilValue(channelsState)
+  const kindHead = policyKind.split('Policy')[0].toLowerCase()
+
+  return (
+    <PageSection>
+      {policies && policies.length > 0 && (
+        <Box mb={3} maxWidth={500}>
+          <DiscoveredViolationsCard
+            policyKind={policyKind}
+            policyViolationSummary={policyViolationSummary(policies ?? [])}
+          />
+        </Box>
+      )}
+      <AcmTable<DiscoveredPolicyItem>
+        id={`${policyKind}ByCluster`}
+        columns={ByClusterCols(
+          t,
+          helmReleases,
+          subscriptions,
+          channels,
+          kindHead == 'operator'
+            ? [
+                {
+                  header: t('Deployment available'),
+                  cell: (item: DiscoveredPolicyItem) => {
+                    if (item.deploymentAvailable == null || item.deploymentAvailable == undefined) return '-'
+                    return item.deploymentAvailable ? 'yes' : 'no'
+                  },
+                  sort: 'deploymentAvailable',
+                  search: 'deploymentAvailable',
+                  id: 'deploymentAvailable',
+                  exportContent: (item: DiscoveredPolicyItem) => {
+                    if (item.deploymentAvailable == null || item.deploymentAvailable == undefined) return '-'
+                    return item.deploymentAvailable ? 'yes' : 'no'
+                  },
+                },
+                {
+                  header: t('Upgrade available'),
+                  cell: (item: DiscoveredPolicyItem) => {
+                    if (item.upgradeAvailable == null || item.upgradeAvailable == undefined) return '-'
+                    return item.upgradeAvailable ? 'yes' : 'no'
+                  },
+                  sort: 'upgradeAvailable',
+                  search: 'upgradeAvailable',
+                  id: 'upgradeAvailable',
+                  exportContent: (item: DiscoveredPolicyItem) => {
+                    if (item.upgradeAvailable == null || item.upgradeAvailable == undefined) return '-'
+                    return item.upgradeAvailable ? 'yes' : 'no'
+                  },
+                },
+              ]
+            : []
+        )}
+        keyFn={(item) => item.cluster}
+        items={policies ?? []}
+        emptyState={
+          <AcmEmptyState
+            title={t(`You don't have any {{kindHead}} policies`, { kindHead })}
+            message={t('The search results returned no {{kindHead}} policies', { kindHead })}
+          />
+        }
+        showExportButton
+        exportFilePrefix={`${policyKind}ByCluster`}
```

Comment:
Should this also have the policy name in the file name?

Reply:
how about tbis `${policyKind}ByCluster-${policyName}`

Reply:
I'm fine with that, but I'd prefer `${policyKind}-${policyName}-all-clusters`

Reply:
good!